README.md

```diff
@@ -51,6 +51,15 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+
+### Experimental features
```

Comment:
Nit: This is currently a subsection under "Contributing". Could we please move this above the "References section"? And use "##" so it is not a subsection?